Add missing script references for failover groups

btasdoven opened the pull request:
There a few failover group cmdlets used in the example script but the script explanation doesn't have links for them. I am adding those missing links.

azurecla commented:
Hi __@v-aljenk__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-aljenk). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

btasdoven committed:
Add missing script references for failover groups

There a few failover group cmdlets used in the example script but the script explanation doesn't have links for them. I am adding those missing links.
